.github/workflows/tox_tests.yml

```diff
@@ -0,0 +1,87 @@
+name: Tox tests on SDK packages
+
+on: [push, pull_request]
+
+jobs:
+  linux-tests:
+
+    runs-on: ${{ matrix.os }}
+    strategy:
+      max-parallel: 4
+      matrix:
+        os: [ubuntu-latest]
+        tox-env: [py27-mylinux, py36-mylinux, py37-mylinux, py38-mylinux]
+        include:
+          - tox-env: py27-mylinux
+            python-version: 2.7
+          - tox-env: py36-mylinux
+            python-version: 3.6
+          - tox-env: py37-mylinux
+            python-version: 3.7
+          - tox-env: py38-mylinux
+            python-version: 3.8
+    env:
+      TOXENV: ${{ matrix.tox-env }}
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install Tox and any other packages
+      run: python -m pip install --upgrade pip setuptools tox
+    - name: Test with tox ${{ matrix.tox-env }}
+      run: python -m tox
+
+  mac-tests:
+
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [macos-latest]
+        tox-env: [py37-mymacos, py38-mymacos]
```

Comment:
I did not notice this earlier, don't we need to add py36-macOS and py36-mywin env? Also py27 seems to be added only on linux any particular reason?

Reply:
To be honest, we can drop the support for `py27` since python 2.7 is EOL already. We should probably suggest users to move out of Python 2.7. I guess until AWS Lambda removes the support, we should support it as well. 

`py36` support probably is on the same lines when checked on different OS'es. Also this is the earlier travis config that we were using before, which worked fine for us. I don't think we need additional configuration at this moment. Maybe we can add new configs in future if we find issues.